transformations/disability_transformation/transformation.py

```diff
@@ -0,0 +1,151 @@
+from interfaces.SentenceOperation import SentenceOperation
+from evaluation.evaluation_engine import evaluate, execute_model
+from tasks.TaskTypes import TaskType
+import spacy
+import string
+from spacy.lang.en.examples import sentences 
+
+# A dict containing offensive words and their alternatives
+disability_names =  {"blind":"person or people with a visual impairment", 
+                           "deformed": "person or people with a physical disability", 
```

Comment:
One suggestion to make your PR stronger, I think in the cases in which you can determine the tense of the subject, you can probably choose just one between "person.." or "people.."? If you are interested you can do that in a separate PR..